distributionviewer/core/static/css/legend.styl

```diff
@@ -0,0 +1,42 @@
+@import 'lib';
+
+$li-margin-bottom = $med-padding;
+
+#legend {
```

Comment:
Not a fan of targeting IDs since they add too much specificity - in case we want to override anything later.